### PR TITLE
[CAT-1522] Adds Cypress and initial healthcheck test

### PR DESCRIPTION
**Pull request by a6software**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -121,6 +122,58 @@ jobs:
           destination: integration/geb
       - store_test_results:
           path: integration-tests/build/test-results
+  integration_tests_cypress:
```
Comment (garethmdavies) [inline .circleci/config.yml:125]: We may want to set the resource class depending on how this build executor runs. Worth keeping an eye on.

File: integration_tests/mockApis/tokenVerification.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+export default {
+  // stubTokenVerificationPing: (status = 200): SuperAgentRequest =>
```
Comment (garethmdavies) [inline integration_tests/mockApis/tokenVerification.ts:5]: Should this be commented out still?

**Review by garethmdavies — DISMISSED** [review]
Looks ok, just a couple of comments

File: integration_tests/mockApis/tokenVerification.ts (hunk 2 above)
Comment (a6software) [reply integration_tests/mockApis/tokenVerification.ts:5]: it comes in the next PR - I must have realised trying to test login in the same PR as getting cypress itself up and running was a touch ambitious 

https://github.com/ministryofjustice/offender-categorisation/pull/694/files#diff-78a88cd5054960f522896487b9a0a3848574cce32f10efe54a475b1b66be7ffe

Comment (a6software) [reply integration_tests/mockApis/tokenVerification.ts:5]: (removing)

**Review by garethmdavies — APPROVED** [review]
approved

**Pull request merged**